src/test/java/bio/terra/common/TestUtils.java

```
@@ -230,24 +230,44 @@ public static TableResult selectFromBigQueryDataset(
   }
 
   public static <T> T mapFromJson(String content, Class<T> valueType) throws IOException {
+    if (content == null) {
+      return null;
+    }
     try {
       return objectMapper.readValue(content, valueType);
-    } catch (IOException ex) {
+    } catch (Exception ex) {
```

IMO we should remove these methods that wrap `readVaule()` and instead let the exceptions be thrown out of our methods. These exceptions will fail the test, which is what we want. Logging here is unnecessary as the top level catch will log the exception and print a stack crawl.

---

I guess.  This does reduce some boiler plate in calling methods though (e.g. not need objects mappers.  I like the idea of centralizing the logic

---

Having a `mapFromJson()` method is fine, the thing I'd change is to remove the `try`/`catch` handler around the call to `objectMapper.readValue()`.

---

Fair enough.  I'll remove the try catch block

---

Actually, now I remember the reason I added the try catch block: it was to generically show the raw JSON that failed to parse for easier debugging.  I'm going to leave